packages/procaptcha/src/modules/ProsopoCaptchaApi.ts

```diff
@@ -60,6 +60,11 @@ export class ProsopoCaptchaApi implements ProsopoCaptchaApiInterface {
 				this.userAccount,
 				this.provider,
 			);
+
+			if (captchaChallenge[ApiParams.error]) {
```

Comment:
why is this needed? It only skips the http->https code

Reply:
because without the return the `captchas` key doesn't exist on the response so the following loop would fail. The API can now return a response with an `error` key, which, when encountered, is displayed in the widget as a red error message.